src/components/chip-set/chip-set.tsx

```diff
@@ -330,7 +330,7 @@ export class ChipSet {
                     {this.label}
                 </label>
                 <div class="mdc-line-ripple" />
-            </div>
+            </label>
```

Comment:
Why change this to a `label`? There is already a label on line 320, which is inside this, and I'm not sure it's valid to nest label elements. This label would also include all the chips, and their respective remove-buttons, something [MDN says we shouldn't do](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Accessibility_concerns).

Reply:
Ah, just read the commit message:

> This makes the HTML structure consistent, like mdc-text-field. They are also wrapped in <label>s. This way we can also apply the same styles to both.

I think it's more important to create well formed and accessible html than to make styling more convenient for us.

Reply:
Do you recommend to change it back to div? @adrianschmidt

Reply:
In the absence of any real recommendation, other than "don't make it a label", I guess so 😄